wagtail_localize/test/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 4.0.5 on 2022-06-05 11:41
+# Generated by Django 4.1.1 on 2022-09-23 14:13
```

Comment:
This isn't a problem with this PR (if it was actually causing a problem, a test would have failed), but just while I am reading this line, a quick note.

For django packages, the recommended practice is to always generate your migrations with the lowest version of django your package is compatible with (which in this case would be 3.2). See https://docs.djangoproject.com/en/4.1/topics/migrations/#supporting-multiple-django-versions for explanation.

Reply:
(this matters even less if the migration is for the test app :smile: )

Reply:
it doesn't matter as I've made the necessary modifications for the initial migration manually